### ISSUE TITLE
111

### ISSUE THREAD

**Issue by Sirius222** [Sirius222/spll#2]
There are some problems with the code. 
Why do you choose 80% of the data in ascending order？
specifically， -> def distilling() methods
def distilling(model,train_loader,train_data):

https://github.com/Hao-Ning/MEIDTM-Instance-Dependent-Label-Noise-Learning-with-Manifold-Regularized-Transition-Matrix-Estimatio/blob/ef1493fcbca4c3fb8c4c98c542eae5d923cb3a29/run_ours.py#L72

also 
Why are the transition matrix network(Linear) parameters not randomly initialized?
